DigitalExcellence/dex-backend
# fixed messy migrations and seeding

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.
1. delete the database
2. run the api
3. check the api for seeded values.

## Link to issue

Closes: #134
